### PR TITLE
ci: trigger repository_dispatch event on release

### PR DESCRIPTION
**Pull request by debdutdeb**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -619,3 +619,18 @@ jobs:
...
+  trigger-dependent-workflows:
+    runs-on: ubuntu-latest
+    needs:
+      - services-docker-image-publish
+      - docker-image-publish
+    steps:
+      - name: Send context to seperate pipeline
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.EXTERNAL_RELEASE_TOKEN }}
```
Comment (debdutdeb) [inline .github/workflows/ci.yml:632]: I named this rather randomly, as idk how secrets are named or set up. Would appreciate some guidance here 🙏🏼

Comment (debdutdeb) [reply .github/workflows/ci.yml:632]: ah forgot to add, need a token with content write permission on [RocketChat/Release.Distributions](https://github.com/RocketChat/Release.Distributions) repo.

Comment (geekgonecrazy) [reply .github/workflows/ci.yml:632]:
```suggestion
          token: ${{ secrets.DISTRIBUTION_TOKEN }}
```

**Review by geekgonecrazy — DISMISSED** [review]

File: .vscode/launch.json
Comment (sampaiodiego) [inline .vscode/launch.json:1]: any reason do include this file?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -619,3 +619,18 @@ jobs:
...
+      - services-docker-image-publish
+      - docker-image-publish
+    steps:
+      - name: Send context to seperate pipeline
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.DISTRIBUTION_TOKEN }}
+          event-type: new_release
+          repository: RocketChat/Release.Distributions
+          client-payload: ${{ format('{"tag":"{0}"}', github.ref_name) }}
```
Comment (sampaiodiego) [inline .github/workflows/ci.yml:635]: this is simpler and easier to read =)
```suggestion
          client-payload: '{"tag": "${{ github.ref_name }}"}'
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

File: .vscode/launch.json
Comment (debdutdeb) [reply .vscode/launch.json:1]: ah .. that sneaked in 🙈 I can remove if you want

Comment (sampaiodiego) [reply .vscode/launch.json:1]: yes pls 😬

Comment (debdutdeb) [reply .vscode/launch.json:1]: :" )

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**